### DISCUSSION TITLE
Como utilizar os arquivos README.md ao seu favor

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#107] [Talks in Portuguese]
**Palestrante:** Leticia Pegoraro Garcez

**Assunto:**
Entenda um pouco mais sobre a importância dos arquivos README.md, e como estruturar estes arquivos para melhorar a apresentação dos seus projetos e dos seu perfil.

**Comment by renata-amorim** [comment]
Leticia, muito obrigada pela palestra, foi muito esclarecedora, estou fazendo migrando de carreira de Nutrição para a área de tecnologia, mais precisamente front-end, e gostaria de saber se você tem um material para indicar além o da palestra para usar melhor o github, o meu está uma bagunça, por exemplo, nunca fiz um READNE.md, já pensei em apagar tudo e começar do 0, enfim.

Parabéns pela palestra e mais uma vez muito obrigada!

**Reply by beatavernaro** [reply]
Oi Renata!
Um video que me ajudou muito a arrumar e até entender sobre markdown pra poder personalizar sozinha foi o da Rafa Ballerini 
[Aqui](https://www.youtube.com/watch?v=TsaLQAetPLU&t=135s&ab_channel=RafaellaBallerini)
Você também pode procurar perfis legais e ver o Readme.md no modo raw pra conseguir visualizar o código por trás! :D

**Reply by renata-amorim** [reply]
Muito obrigada 😄

**Reply by biawenzel** [reply]
@beatavernaro muito obrigada, eu tenho o mesmo problema que a @renata-amorim , tbm estou entrando na area agora e nunca fiz um README, vou assistir o video que vc indicou!

**Reply by LelePG** [reply]
Oi @renata-amorim  e @biawenzel , tenho alguns links que coloquei no material extra que está no comentário abaixo. 
Ali tem alguns links de vários repositórios com recursos interessantes pra vocês usarem no perfil de vocês, e nesse material também tem o link de dois artigos que eu escrevi que também falam um pouco sobre como fazer READMEs.

Essa dica da @beatavernaro de olhar o README de perfil de perfis que tu acha interessante no modo raw é maravilhosa. Inclusive, tem um repositório aqui que tem uma lista de repositórios de README de perfil que podem te dar algumas ideias: [Awesome Github READMEs](https://zzetao.github.io/awesome-github-profile/), infelizmente ele está em inglês, como muitos repositórios que eu coloquei no material extra (indico a instalação da extensão do google tradutor caso tu não tenha muito domínio de inglês, ela não é perfeita, mas ajuda).

Vou deixar aqui também o link de uma playlist com alguns vídeos que eu fiz revisando os perfis no Github de colegas de trabalho meus. Tem várias dicas nessa série que talvez possam te ajudar também: [Playlist de Github](https://www.youtube.com/watch?v=DsgM3GGY-wM&list=PLdPPE0hUkt0qzX9wma3p6Gq4TOl0beV29&index=3)

Recomendo também que vocês dêem uma stalkeada nos perfis de algumas pessoas pra ver como que elas estão estruturando os READMEs delas. Vejam o perfil, vejam os projetos e filtrem o que vocês acham que é relevante para os projetos/perfil de vcs 😄

**Reply by renata-amorim** [reply]
Muito obrigada Lê ❤️

**Comment by LelePG** [comment]
Link do material extra que eu falei na palestra : [clique aqui](https://www.canva.com/design/DAFKp5vYDkA/hlL1iyYYFqK6ObzaufrAWg/view?utm_content=DAFKp5vYDkA&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)

Minhas redes: 
Github: lelepg
LinkedIn: lelepg
e-mail: lpgarez@inf.ufpel.edu.br
Obs: sintam-se a vontade para entrar em contato comigo, só preciso avisar que não entro muito no LinkedIn, então o e-mail é a forma mais rápida de falar comigo
😄

**Comment by biawenzel** [comment]
Muito obrigada, Lê!!

Beatriz Wenzel Pereira.


Em qui., 15 de set. de 2022 às 14:55, Renata ***@***.***>
escreveu:
